src/index.html

```diff
@@ -15,8 +15,73 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          class="imagem"
+          src="./images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+      <nav class="links">
+        <a
+          href="#Apple"
+          class="is-active link"
+        >
+          APPLE
+        </a>
+        <a
+          href="#Samsung"
+          class="link"
+        >
+          SAMSUNG
+        </a>
+        <a
+          href="#Smartphones"
+          class="link"
+        >
+          SMARTPHONES
+        </a>
+
+        <a
+          href="#laptops-computers"
+          class="link"
+          data-qa="hover"
```

Comment:
Don't use spaces in `<a>` tag's `href` property. The `href` for 'LAPTOPS & COMPUTERS' should be '#laptops-computers' without spaces.